src/components/Image/docs/examples.ts

```diff
@@ -17,3 +17,13 @@ export const relativeUriExample = `
   alt="Garfield smiles and puts his hand over chest"
 />
 `;
+
+export const blurLoadingExample = `
```

Comment:
consider adding a button to retrigger the rendering since it's very quick.

Reply:
Basically you can just change any value in the props to see it again, but I'll add a button to make it easier 👌